### ISSUE TITLE
choose the venv with a custom dynamic inventory script

### ISSUE THREAD

**Issue by Mikey032** [ansible/awx#4186]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY

We are pulling data from a mysql database to create an inventory. This script uses the mysql-connector to connect to the mysql database. We already have a custom venv mounted inside every container with all our needed python modules and we would also like to use this custom venv in our custom dynamic inventory scripts. This would allow us to not need to rebuild the awx task and web containers.

**Comment by ryanpetrello** [comment]
@Mikey032,

Can you clarify the request a bit here?  Are you wanting to select the custom virtualenv in the custom script editing form?

**Comment by Mikey032** [comment]
Yes indeed. Choosing a custom virtualenv in the custom script will allow us to not modify the docker images and mount us a custom virtualenv inside the container that the custom script can use.

**Comment by chris93111** [comment]
+1
